doc/components/pgr_makeConnected.rst

```diff
@@ -7,6 +7,13 @@
     Alike 3.0 License: http://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+
+* **Supported versions:**
+  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_makeConnected.html>`__)
```

Comment:
Also add the latest
```
`Latest <https://docs.pgrouting.org/latest/en/pgr_makeConnected.html>`__)
```

Reply:
the link does not work

Reply:
Add the missing line
```
`Latest <https://docs.pgrouting.org/latest/en/pgr_makeConnected.html>`__
```
and on this file: doc/conf.py.in around line 236 we have a list of links that might not be working.
https://github.com/pgRouting/pgrouting/blob/develop/doc/conf.py.in#L326

```
'https://docs.pgrouting.org/lastes/en/pgr_makeConnected.html' ,
```
Use normal ' and put a comma at the end

Reply:
Latest goes before the 3.2